src/wallet/wallet.h

```diff
@@ -579,11 +579,10 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
         SetNull();
     }
 
-    CWallet(const std::string& strWalletFileIn)
+    CWallet(const std::string& strWalletFileIn) : strWalletFile(strWalletFileIn)
```

Comment:
Yes, I believe it's easier to read if the initialization is separated from the constructor prototype.

> Take a look at wallet.h as a whole.

Not sure what you're referring to.

RE: Travis. There seems to be quite a bit of non-determinism in the travis tests.  It's something we're looking into but we don't have a solution for that problem yet.